models/fates/inst/create_CLM5-FATES_1x1br_refrun.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+#
+#
+#=============================================================================================
+#
+# Generate CLM5-FATES reference run for PEcAn site-scale simulations
+# 
+# Users should update all of the lines with -USER UPDATE to match their local environment
+#
+# --- Last updated by Shawn P. Serbin <sserbin@bnl.gov>
+#=============================================================================================
+
+# Define CTSM-FATES options
+export MODEL_SOURCE=/data/software/CTSM/ctsm                   	# CTSM/CLM/ELM source directory - USER UPDATE
+export PROJECT=PEcAn_simulations                               	# project or run tag
+export COMPSET=I2000Clm50FatesGs				# define compset for run
+export RES=1x1_brazil						# set grid resolution
+export MAC=modex                                             	# Name your machine - USER UPDATE
```

Comment:
@serbinsh Is the modex machine settings a default in the machine config file or did you make your own machine settings?

Reply:
Shoot, that needs to be switched back to a default machine name there like before and the end user needs to then either leave it or switch it to match their local machine.  I will switch it back to eddi, which is what it was before

Reply:
Cool! At some point could I see your machine settings? I don't think eddi would work for us. Also, in the future we'll need to make some documentation on how a user would update these settings.